lib/kitchen/templates/eoc_section_title_template.xhtml.erb

```diff
@@ -3,7 +3,7 @@
     <span class="os-text"><%= @title %></span>
   </<%= @in_composite_chapter ? 'h3' : 'h2' %>>
```

Comment:
This can be DRYed up a hair

```ruby
<%- main_title_tag = @in_composite_chapter ? 'h3' : 'h2' 
-%>
<div class="os-eoc os-<%=@klass%>-container" data-type="composite-page" data-uuid-key="<%=@uuid_prefix%><%=@klass%>">
  <<%= main_title_tag %> data-type="<%= @in_composite_chapter ? 'title' : 'document-title' %>">
    <span class="os-text"><%= @title %></span>
  </<%= main_title_tag %>>